pcdsdevices/areadetector/detectors.py

```diff
@@ -383,6 +406,10 @@ def _open_screen(self):
     def _open_screen(self, value):
         self.open_viewer()
 
+    def screen(self):
+        """ Lean on open_viewer method """
+        self.open_viewer()
```

Comment:
Ah, didn't see this before making previous comment

Reply:
Yeah, I didn't want to remove this in case I was missing an edge-case where they behaved differently.  To my knowledge, they should be identical.